### PR TITLE
✨ FEAT : LIVE-4021 - Learn Content in LLM

### PR DESCRIPTION
**Pull request by mcayuelas-ledger**
### 📝 Description

New Learn Content in LLM with content cards

### ❓ Context

- **Impacted projects**: `live-mobile` `ui:native` 
- **Linked resource(s)**: [LIVE-4021] 

### ✅ Checklist

- [ ] **Test coverage** 
- [x] **Atomic delivery** 
- [x] **No breaking changes** 

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

[LIVE-4021]: https://ledgerhq.atlassian.net/browse/LIVE-4021?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/ledger-live-mobile/src/screens/Learn/learn.tsx [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+        renderItem={renderItem}
+        keyExtractor={keyExtractor}
+        showsVerticalScrollIndicator={false}
+      />
+    </Flex>
+  );
+}
+
+const Container = styled(TouchableHighlight)`
+  padding-left: 16px;
```
Comment (nparigi-ledger) [inline apps/ledger-live-mobile/src/screens/Learn/learn.tsx:82]: Use spacing values

File: libs/ui/packages/native/storybook/stories/Cards/Cards.stories.tsx [hunk 2]
```diff
@@ -94,5 +126,13 @@ storiesOf((story) =>
           component: descriptionCardB,
         },
       },
+    })
+    .add("Card D", CardDStory, {
```
Comment (nparigi-ledger) [inline libs/ui/packages/native/storybook/stories/Cards/Cards.stories.tsx:130]: rename "cardD" with the new name of the card ?

File: apps/ledger-live-mobile/src/screens/Learn/index.tsx [hunk 3]
```diff
@@ -1,41 +1,102 @@
...
+              variant="small"
+              fontWeight="medium"
+              color="neutral.c70"
+              ml={2}
+            >
+              ledger.com/academy
+            </Text>
+          </Flex>
+          <Flex width="20%" alignItems="flex-end">
+            <TouchableOpacity onPress={goBack} style={{ padding: 16 }}>
```
Comment (nparigi-ledger) [inline apps/ledger-live-mobile/src/screens/Learn/index.tsx:82]: use theme values for padding

File: apps/ledger-live-mobile/src/screens/Learn/learn.tsx [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+    },
+    [navigation, trackContentCardEvent],
+  );
+
+  const renderItem = ({ item: card }: { item: LearnContentCard }) => {
+    return (
+      <Skeleton
+        loading={isLoading}
+        height="85px"
+        mx="16px"
```
Comment (nparigi-ledger) [inline apps/ledger-live-mobile/src/screens/Learn/learn.tsx:50]: use theme values for margin and borderRadius

**Review by cgrellard-ledger — APPROVED** [review]

**Pull request merged**